components/courses/app/models/courses/student.rb

```diff
@@ -25,5 +25,7 @@ class Student < ApplicationRecord
     validates :personal_info,   presence: true
     validates :motivation_info, presence: true
     validates :user_id,         uniqueness: { scope: :season_id }
+
+    scope     :attending, -> { where(status: :attending) }
```

Comment:
I am sure that `ActiveRecord::Enumerator` creates scopes for you for free and you shouldn't duplicate it